index.html

```diff
@@ -13,7 +13,8 @@
         // IE11 and earlier can be identified via their user agent and be
         // redirected to a page that is known to have no newer js syntax.
         if (window.navigator.userAgent.match(/(MSIE|Trident)/)) {
-            window.location.href = "static/recommendedBrowsers.html";
+            var roomName = encodeURIComponent(window.location.pathname);
```

Comment:
This won't work with tenants. A URL like so: `https://meet.jit.si/saghul/test123` yields `/saghul/test123` as the pathname. You probably want to find the `lastIndexOf` `/` and take the last part.

Reply:
Actually, we want to take the tenant as well, the meeting name will be /saghul/test123. This is the expected name, tested on chrome and it is the same.

Reply:
aha, then we are good